src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -53,44 +54,39 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
     };
   }
 
-  /**
-   * Ignore this hard-coded method.
-   * This will be updated in a separate PR.
-   */
   getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    return [['orgStats', this.endpointPath, {query: this.endpointQuery}]];
+  }
+
+  get endpointPath() {
     const {organization} = this.props;
+    return `/organizations/${organization.slug}/stats_v2/`;
+  }
 
-    return [
-      [
-        'orgStats',
-        `/organizations/${organization.slug}/stats_v2/`,
-        {
-          query: {
-            interval: '1d',
-          },
-        },
-      ],
-    ];
+  get endpointQuery() {
+    const {dateStart, dateEnd} = this.props;
+    return {
+      statsPeriod: `${dateEnd.diff(dateStart, 'd')}d`, // TODO(org-stats)
+      interval: '1d', // TODO(org-stats)
+      groupBy: ['category', 'outcome'],
+      field: ['sum(quantity)', 'sum(times_seen)'],
+    };
   }
 
   get chartMetadata() {
```

Comment:
wondering if MetaData is a bit of a misnomer since it actually contains all of the data the chart will use. wonder if this func named chartData would be better, then the chartData becomes chartSeriesData?

Reply:
Yep sure, I'll amend this later in the branch `dlee/org-stats-7` to avoid merge conflicts.